lib/menu.rb

```diff
@@ -0,0 +1,11 @@
+class Menu
+  def initialize
+    @dishes = {"margherita" => 4.50, "al funghi" => 5.50, "hot & spicy" => 6.00, "calzone" => 7.50, "hawaiian" => 6.50, "chicken supreme" => 9.50, "garlic bread" => 3.00}
```

Comment:
Metrics/LineLength: Line is too long. [170/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)